app/models/user.rb

```diff
@@ -42,8 +42,9 @@ def self.create_with_omniauth(auth)
 
   # Activates an account.
   def activate
-    update_attribute(:activated, true)
-    update_attribute(:activated_at, Time.zone.now)
+    self.activated = true
+    self.activated_at = Time.zone.now
+    save!
```

Comment:
I think this is better - I think activation *should* change the updated_at field.